README.md

```diff
@@ -122,7 +124,19 @@ beforeAll(() => {
 });
 ```
 
-That's it! You're good to go.
+### With Karma and TypeScript in an Angular project
+
+Add to the existing `types` array in `tsconfig.spec.json`: `"@testing-library/jasmine-dom"`
+
+In your tests setup file, import jasmine-dom like so:
```

Comment:
In your tests setup file (`test.ts` in an Angular project), import jasmine-dom and add to the Jasmine matchers like so: